Build fails in linux "AskSaveCheat" was not declared in this scope

Building on ubuntu 18.04, witih g++, after commit fb1d489cfaabe19789d0444704b81e55ec7d2fc4 I get the following error when running scons.

```
src/fceu.cpp: In function 'void FCEU_CloseGame()':
src/fceu.cpp:195:5: error: 'AskSaveCheat' was not declared in this scope
     AskSaveCheat();
     ^~~~~~~~~~~~
src/fceu.cpp:195:5: note: suggested alternative: 'savecheats'
     AskSaveCheat();
     ^~~~~~~~~~~~
     savecheats
```

--- a/src/fceu.cpp
+++ b/src/fceu.cpp
@@ -189,11 +189,13 @@ static void FCEU_CloseGame(void)
 		}
 
 		if (GameInfo->type != GIT_NSF) {
+#ifdef WIN32
 			if (disableAutoLSCheats == 2)
 				FCEU_FlushGameCheats(0, 1);
 			else if (disableAutoLSCheats == 1)
 				AskSaveCheat();
 			else if (disableAutoLSCheats == 0)
+#endif
 				FCEU_FlushGameCheats(0, 0);
 		}
 

